cobalt/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -769,6 +769,12 @@ public boolean start() {
 
   @SuppressWarnings("unused")
   @UsedByNative
+  private boolean restart() {
+    // Restart MediaCodec after flush().
```

Comment:
It looks like there isn't a functional different between `start()` and `restart()` other than possibly printing a error. Should we add a log here to indicate that the mediacodec is restarting?